connector-splunk/src/main/java/com/redhat/cloud/notifications/connector/splunk/eventing/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * This package contains classes that were imported from the eventing-integrations
+ * repository with no significant changes made to the original source code.
+ * Major changes will come as part of an upcoming refactoring, after the Splunk
+ * connector has been successfully deployed and tested in production.
+ */
```

Comment:
This should be read before reviewing any class from the `com.redhat.cloud.notifications.connector.splunk.eventing` package.